apps/src/templates/projects/ProjectCardGrid.story.jsx

```diff
@@ -33,33 +30,72 @@ function generateFakeProject(overrideData) {
   };
 }
 
-function generateFakePublicProjects() {
+function generateFakePublicProjectsWithStudentInfo() {
   let date = new Date();
-  let publicProjects = [];
+  let publicProjects = {};
   let i = 0;
     projectTypes.forEach(type => {
+      publicProjects[type] = [];
       for (i=0; i < 5; i++) {
-        publicProjects.push(generateFakeProject({type: type, projectName: type, updatedAt: date.setDate(date.getDate() - i ), studentAge: 6 + 3*i}));
+        publicProjects[type].push(generateFakeProject({
+          type: type,
+          name: type,
+          publishedAt: new Date(date.getTime() - i * 60 * 1000).toISOString(),
+          studentName: 'Penelope',
+          studentAge: 6 + 3 * i
+        }));
       }
     });
   return publicProjects;
 }
 
+function generateFakePublicProjectsWithoutStudentInfo() {
+  let date = new Date();
+  let publicProjects = {};
+  let i = 0;
+  projectTypes.forEach(type => {
+    publicProjects[type] = [];
+    for (i=0; i < 5; i++) {
```

Comment:
nit (and feel free to ignore) - i tend to find these sorts of things to be easier to read if I do it more functionally, i.e.:
```
publicProjects.type = _.range(5).map(index => {
   // etc
});
``